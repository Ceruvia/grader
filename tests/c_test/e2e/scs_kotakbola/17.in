88 825 876
H 21
H 46
E
