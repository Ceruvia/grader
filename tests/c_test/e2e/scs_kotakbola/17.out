67 0 0
Ya
